portfolio.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <title>About</title>
+        <link rel="stylesheet" href="css/styles.css">
+    </head>
+    <body>
+     
+      
+      
+      
+      <!-- ************ PROJECTS / PORTFOLIO  ************** -->
+      <section class="projects">
+        <div class="content-wrap divider">
+          <h2>Featured Projects</h2>
+          <p>View selected projects below. More information can be found at <a href="https://github.com/julianadecarvalho">Juliana's Github Repo</a>.</p>
+
+          <!-- Project 1 -->
+          <section class="project-item">
+            <img src="images/swap-meet.jpeg" alt="Swap Meet">
```

Comment:
The image is a `png` in your images folder
```suggestion
            <img src="images/swap-meet.png" alt="Swap Meet">
```